datacapture/src/test/java/com/google/android/fhir/datacapture/validation/MinValueConstraintValidatorTest.kt

```diff
@@ -55,6 +56,29 @@ class MinValueConstraintValidatorTest {
     assertThat(validationResult.message).isEqualTo("Minimum value allowed is:10")
   }
 
+  @Test
+  fun shouldReturnInvalidResultForDecimal() {
```

Comment:
```suggestion
  fun shouldReturnInvalidResult() {
```

so in terms of responsibilities, the `MinValueConstraintValidator` only checkes if something is less than the min value. It doesn't really care what type the value is. So in the tests, what we are testing is that it invokes the comparison correctly.

On the other hand, the `MoreTypeTest` needs to verify the comparison works with different types of value.

So here I think it makes more sense to remove the `ForDecimal` in the name of the test case.